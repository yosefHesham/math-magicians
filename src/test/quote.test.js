import React from 'react';
import renderer from 'react-test-renderer';
import Quote from "../pages/quote.js"

describe('Test for quote page', () => {
  test('Quote page should render correctly', () => {
    const quote = renderer
      .create(<Quote/>)
      .toJSON();
    expect(quote).toMatchSnapshot()
  });
});