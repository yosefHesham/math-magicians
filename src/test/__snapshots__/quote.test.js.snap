// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for quote page Quote page should render correctly 1`] = `
<div
  className="quote"
>
  Mathmatics is not about numbers, equations, computations, or algorithms: its about understanding. - William Paul Thurston
</div>
`;
